products/BellHybrid/apps/application-bell-onboarding/windows/OnBoardingShortcutsWindow.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "OnBoardingShortcutsWindow.hpp"
+#include "BellOnBoardingNames.hpp"
+
+#include <Style.hpp>
+#include <SideListView.hpp>
+#include <gui/input/InputEvent.hpp>
+#include <keymap/KeyMap.hpp>
+
+namespace gui
+{
+    OnBoardingShortcutsWindow::OnBoardingShortcutsWindow(
+        app::ApplicationCommon *app,
+        std::unique_ptr<app::OnBoarding::OnBoardingShortcutsWindowContract::Presenter> &&presenter,
+        const std::string &name)
+        : AppWindow(app, name), presenter{std::move(presenter)}, application{app}
```

Comment:
You probably don't need `application` since it's already stored in base class `AppWindow`.

Reply:
Actually, I need `application` in this file because I use it to `render` and `switchWindow` :)

Reply:
My point was that app is stored in `AppWindow` and is a protected member so is accessed from derived class. There is also a member function returning it.

Reply:
Done